### PR TITLE
Change/add agent to server

### PR DESCRIPTION
**Pull request by johntdyer**
Change/add agent to server

### CODE REVIEW

File: thousandeyes/resource_agent_to_server.go [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+	}
+
+	if d.HasChange("bgp_measurements") {
+		update.BgpMeasurements = d.Get("bgp_measurements").(int)
+	}
+	if d.HasChange("agents") {
+		update.Agents = expandAgents(d.Get("agents").([]interface{}))
+	}
+
+	// Need to figure this out
```
Comment (william20111) [inline thousandeyes/resource_agent_to_server.go]: can we create an issue to track to track this, just so it doesnt get lost and forgotten about.

File: thousandeyes/resource_agent_to_server.go [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+		return err
+	}
+	d.SetId("")
+	return nil
+}
+
+func resourceAgentServerCreate(d *schema.ResourceData, m interface{}) error {
+	client := m.(*thousandeyes.Client)
+	log.Printf("[INFO] Creating ThousandEyes Test %s", d.Id())
+	webTrans := buildAgentServerStruct(d)
```
Comment (william20111) [inline thousandeyes/resource_agent_to_server.go]: can we clean up the var names here

File: thousandeyes/resource_agent_to_server.go (hunk 1 above)
Comment (johntdyer) [reply thousandeyes/resource_agent_to_server.go]: https://github.com/william20111/terraform-provider-thousandeyes/issues/7

File: thousandeyes/resource_agent_to_server.go (hunk 2 above)
Comment (johntdyer) [reply thousandeyes/resource_agent_to_server.go]: fixed

**Comment by johntdyer** [comment]
@william20111  I dont have permissions to merge it myself actually

**Comment by william20111** [comment]
Yes sorry was busy last night will get this merged today and tag a release

**Pull request merged**